### PR TITLE
resource/alicloud_logtail_config: Add last_modify_time parameter to resourceAlicloudLogtailConfig for conditional state refresh

### PR DESCRIPTION
**Pull request by quzard**

### CODE REVIEW

**Review by ChenHanZhang — COMMENTED** [review]
approved

File: alicloud/resource_alicloud_logtail_config.go [hunk 1]
```diff
@@ -122,6 +134,37 @@ func resourceAlicloudLogtailConfigRead(d *schema.ResourceData, meta interface{})
 		return WrapError(err)
 	}
 
+	// Save the input_detail in tf to state, instead of saving the server-side config's input_detail.
+	inputDetail := d.Get("input_detail").(string)
+	lastModifyTimeNew := strconv.Itoa(int(config.LastModifyTime))
+	d.Set("input_detail", inputDetail)
```
Comment (yyuuttaaoo) [inline alicloud/resource_alicloud_logtail_config.go]: 没必要

File: alicloud/resource_alicloud_logtail_config.go [hunk 2]
```diff
@@ -122,6 +134,37 @@ func resourceAlicloudLogtailConfigRead(d *schema.ResourceData, meta interface{})
...
+	d.Set("last_modify_time", lastModifyTimeNew)
+	d.Set("project", split[0])
+	d.Set("name", config.Name)
+	d.Set("logstore", split[1])
```
[the remark this answers is not in the document]
Comment (quzard) [reply alicloud/resource_alicloud_logtail_config.go:142]: DescribeLogtailConfig 检查了长度

**Review by yyuuttaaoo — APPROVED** [review]

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request merged**